api/go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/ramendr/ramen/api
 
-go 1.22.5
+go 1.23.0
 
-toolchain go1.22.7
+toolchain go1.23.5
```

Comment:
According to Boris Ranto:

> toolchain directive is preferred version while go directive sets the minimum required version

So this change is incorrect - if go1.23.5 is not available we will build with older version that may not have the fix.